src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InteractiveWindow_InProc.cs

```diff
@@ -213,5 +215,10 @@ public void WaitForReplOutputContains(string outputText)
                 await Task.Delay(50);
             }
         }
+
+        protected override ITextBuffer GetBufferContainingCaret(IWpfTextView view)
```

Comment:
I admit I don't get what this method is supposed to mean here. The buffer you're returning is the surface buffer which has nothing to do with the caret at all. I suspect a more precise name here is good?